packages/create-neutrino-project/commands/init/generator/prompting.js

```diff
@@ -0,0 +1,23 @@
+const questions = require('./questions');
+
+module.exports = function () {
+  const defaults = {
+    author: '',
+    description: 'Get Started with Neutrino!'
+  };
+  const done = this.async();
+  const command = this.spawnCommand('bash', [
```

Comment:
We shouldn't need this author command if we are going to spawn a command for `yarn init` or `npm init`.